scripts/get_information.py

```diff
@@ -0,0 +1,10 @@
+import requests
+from bs4 import BeautifulSoup
+
+url = "https://www.stonybrook.edu/sb/bulletin/current/academicprograms/cse/courses.php"
+
+response = requests.get(url)
+
+soup = BeautifulSoup(response.content, 'html.parser')
+
```

Comment:
## Notes for later:

Check out this [link](https://stackoverflow.com/questions/2136267/beautiful-soup-and-extracting-a-div-and-its-contents-by-id) to see how you can check the website by divs.